Fixup indentation for docs on `ModuleSpec` attributes (#124681)
python/cpython: Doc/library/importlib.rst

.. class:: ModuleSpec(name, loader, *, origin=None, loader_state=None, is_package=None)

   A specification for a module's import-system-related state.  This is
   typically exposed as the module's :attr:`__spec__` attribute.  Many

>    of these attributes are also available directly on a module: e.g.,

Reviewer: ```suggestion
   of these attributes are also available directly on a module: for example,
```

   .. attribute:: name

      The module's fully qualified name

>       (see :attr:`__name__ attributes on modules <__name__>`).

Reviewer: ```suggestion
      (see :attr:`__name__` attributes on modules).
```

      The :term:`finder` should always set this attribute to a non-empty string.

   .. attribute:: loader

      The :term:`loader` used to load the module

>       (see :attr:`__loader__ attributes on modules <__loader__>`).

Reviewer: ```suggestion
      (see :attr:`__loader__` attributes on modules).
```

      The :term:`finder` should always set this attribute.

   .. attribute:: origin

      The location the :term:`loader` should use to load the module

>       (see :attr:`__file__ attributes on modules <__file__>`).

Reviewer: ```suggestion
      (see :attr:`__file__` attributes on modules).
```

      For example, for modules loaded from a .py file this is the filename.
      The :term:`finder` should always set this attribute to a meaningful value
      for the :term:`loader` to use.  In the uncommon case that there is not one
      (like for namespace packages), it should be set to ``None``.

   .. attribute:: submodule_search_locations

      The list of locations where the package's submodules will be found

>       (see :attr:`__path__ attributes on modules <__path__>`).

Reviewer: ```suggestion
      (see :attr:`__path__` attributes on modules).
```

      Most of the time this is a single directory.
      The :term:`finder` should set this attribute to a list, even an empty one, to indicate
      to the import system that the module is a package.  It should be set to ``None`` for
      non-package modules.  It is set automatically later to a special object for
      namespace packages.

   .. attribute:: loader_state

      The :term:`finder` may set this attribute to an object containing additional,
      module-specific data to use when loading the module.  Otherwise it should be
      set to ``None``.

   .. attribute:: cached

      The filename of a compiled version of the module's code

>       (see :attr:`__cached__ attributes on modules <__cached__>`).

Reviewer: ```suggestion
      (see :attr:`__cached__` attributes on modules).
```

      The :term:`finder` should always set this attribute but it may be ``None``
      for modules that do not need compiled code stored.

   .. attribute:: parent

      (Read-only) The fully qualified name of the package the module is in (or the
      empty string for a top-level module).

>       See :attr:`__package__ attributes on modules <__package__>`.

Reviewer: ```suggestion
      See :attr:`__package__` attributes on modules.
```
